_pytest/skipping.py

```diff
@@ -260,7 +266,7 @@ def pytest_report_teststatus(report):
     if hasattr(report, "wasxfail"):
         if report.skipped:
             return "xfailed", "x", "xfail"
-        elif report.failed:
+        elif report.passed:
```

Comment:
@fabioz I think this will mean you no longer have to do special handling for this in [pydev_runfiles_pytest2.py](https://github.com/fabioz/Pydev/blob/master/plugins/org.python.pydev/pysrc/_pydev_runfiles/pydev_runfiles_pytest2.py#L222), correct?